main.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"go-bootcamp/action"
+	"go-bootcamp/data"
+	"log"
```

Comment:
Just separate the packages using best practices:

Go core

Project Local 

Third-Party